categories.js

```diff
@@ -46,9 +45,7 @@ exports.normalizeGameName = (game) => {
 // Given a game name (normalizedGame) and a category string, returns the closest matching category name.
 exports.normalizeCategory = (normalizedGame, category) => {
     normalizedCategory = category.toLowerCase()
-            .replace(new RegExp(" ", 'g'), "")
-            .replace(new RegExp("-", 'g'), "")
-            .replace("%", "")
+            .replace(/\W/g, "")
```

Comment:
Won't this remove `+`, which could cause "NG+" to not match correctly?

Reply:
Yes, you have to swap the lines

Reply:
The line below just replaces the word "plus" with "+". I mean if someone types in "+", it'll be removed here no matter what.